doc-github-rules.md

```diff
@@ -0,0 +1,106 @@
+  - [Create a Pull Request](#create-a-pull-request)
+- [Create a Pull Request from a branch in the original repository](#create-a-pull-request-from-a-branch-in-the-original-repository)
+- [PR Review Phase](#pr-review-phase)
+- [PR Accept Phase](#pr-accept-phase)
+- [Directly Editing Artifacts](#directly-editing-artifacts)
+- [Use checklists for large issues or PRs](#use-checklists-for-large-issues-or-prs)
+- [Tools](#tools)
+
+This document describes a short process to effectively use GitHub to achieve the following main goals:
+
+1. Proposing a change. 
+2. Reporting a bug. 
+3. Making improvement suggestions.   
+
+> [!NOTE]
+> The intent of this document is not to teach the use of GitHub. The document just gathers some basic rules for a quick reference. 
+
+## Create issues
+
+You use the **Issues** mechanism to report problems, bugs and suggest general improvements. This is for both internal and external contributors. 
+
+This approach is useful if you want to 
+
+- Flag items that cannot be addressed by a single PR.
+- Request that may need to be analyzed to find the proper course of action.
+- Record information to put on backlog.  
+
+Please, include relevant details the rest of the community needs to understand in order to discuss the issue effectively. 
+
+For more information, see [Creating an issue](https://docs.GitHub.com/en/issues/tracking-your-work-with-issues/creating-an-issue). 
+
+
+## Create a fork
+
+You fork a repository to propose changes to the original repository. This is the recommended practice for community contributions.
+
+For more information, see [Fork a repo](https://docs.github.com/en/get-started/quickstart/fork-a-repo).
+
+If you use the Github desktop app, see [Cloning and forking repositories from GitHub Desktop](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/adding-and-cloning-repositories/cloning-and-forking-repositories-from-github-desktop). 
+   
+It's good practice to regularly sync your fork with the original repository. 
+For more information, see [Syncing a fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork).
+
+### Create a Pull Request
+
+You can make any changes to a fork, including making branches and opening pull requests. 
+If you want to contribute to the original repository, you can submit a pull request.
+
+For more information, see [Creating a pull request from a fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork).
+
+If you use the Github desktop app, see [Creating an issue or pull request](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/working-with-your-remote-repository-on-github-or-github-enterprise/creating-an-issue-or-pull-request). 
+
+## Create a Pull Request from a branch in the original repository
```

Comment:
I would prefer we remove this as I would rather we only do PRs from forks and not from the original repository.

Reply:
Done. I agree, that had been a concession to the original project, which uses local branches more than most Community projects.